documentation/examples/github/full.ts

```diff
@@ -0,0 +1,159 @@
+import * as coda from "@codahq/packs-sdk";
+export const pack = coda.newPack();
+
+// Regular expression used to parse repo URLs.
+const RepoUrlRegex = new RegExp("^https://github\.com/([^/]+)\/([^/]+)");
+
+// How many items to fetch per-page when making API list requests.
+const PageSize = 50;
+
+// Allow the Pack to access the GitHub domain.
+pack.addNetworkDomain("github.com");
+
+// Setup per-user authentication using GitHub's OAuth2.
+// Remember to set your client ID and secret in the "Settings" tab.
+// See https://docs.github.com/en/developers/apps/building-oauth-apps
+pack.setUserAuthentication({
+  type: coda.AuthenticationType.OAuth2,
+  authorizationUrl: "https://github.com/login/oauth/authorize",
+  tokenUrl: "https://github.com/login/oauth/access_token",
+  tokenPrefix: "token",
+  scopes: ["repo", "user"],
+
+  // Determines the name of the GitHub account that was connected.
+  getConnectionName: async function (context) {
+    let response = await context.fetcher.fetch({
+      method: "GET",
+      url: "https://api.github.com/user",
+    });
+    return response.body.login;
+  }
+});
+
+// A schema that defines a repo object.
+const RepoSchema = coda.makeObjectSchema({
+  properties: {
+    id: { type: coda.ValueType.Number },
```

Comment:
have we been using extra spaces in curly braces in the ts files in examples? doesn't seem the case in other examples. 

just checking if it's new or old. I think (to me) it's the VS new version defaults to add spaces between curly braces. 

Some settings like this
![image](https://user-images.githubusercontent.com/70549840/152599723-088f7b97-be0d-4457-932b-906d8be447fb.png)

Reply:
I don't think I've done it consistently, but I've been trying to do it more since that's how the Web IDE auto-formats the code. I'll do a pass on the code in a separate PR to standardize this and add a lint rule.